File: AppGenerateFiles/facturacion/Model/Transaccion_Factura.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Transaccion_Factura : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? id_factura { get; set; }
       public string? tipo { get; set; }
       public string? concepto { get; set; }
       public Double? tasa_cambio { get; set; }
       public Double? total { get; set; }
       public int? id_cliente { get; set; }
       public int? id_sucursal { get; set; }
       public DateTime? fecha { get; set; }
       public int? id_usuario { get; set; }
       public string? Factura_contrato { get; set; }
       public string? estado { get; set; }
       public string? no_factura { get; set; }
       public Double? subtotal { get; set; }
       public Double? iva { get; set; }
       [OneToMany(TableName = "Detalle_Factura_Recibo", KeyColumn = "id_factura", ForeignKeyColumn = "id_factura")]
       public List<Detalle_Factura_Recibo>? Detalle_Factura_Recibo { get; set; }
   }
}

File: UI/wwwroot/Facturacion/FrontModel/Transaccion_Factura.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { Detalle_Factura_Recibo }  from './Detalle_Factura_Recibo.js'
class Transaccion_Factura extends EntityClass {
   constructor(props) {
       super(props, 'EntityFacturacion');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ id_factura;
   /**@type {String}*/ tipo;
   /**@type {String}*/ concepto;
   /**@type {Number}*/ tasa_cambio;
   /**@type {Number}*/ total;
   /**@type {Number}*/ id_cliente;
   /**@type {Number}*/ id_sucursal;
   /**@type {Date}*/ fecha;
   /**@type {Number}*/ id_usuario;
   /**@type {String}*/ Factura_contrato;
   /**@type {String}*/ estado;
   /**@type {String}*/ no_factura;
   /**@type {Number}*/ subtotal;
   /**@type {Number}*/ iva;
   /**@type {Array<Detalle_Factura_Recibo>} OneToMany*/ Detalle_Factura_Recibo;
}
export { Transaccion_Factura }
